x-pack/solutions/security/plugins/security_solution/public/app/home/global_header/index.tsx

```diff
@@ -80,7 +81,10 @@ export const GlobalHeader = React.memo(() => {
   const newDataViewPickerEnabled = useIsExperimentalFeatureEnabled('newDataViewPickerEnabled');
 
   const dataViewPicker = newDataViewPickerEnabled ? (
-    <DataViewPicker scope={sourcererScope} />
+    <DataViewPicker
+      scope={sourcererScope}
+      disabled={sourcererScope === DataViewManagerScopeName.detections}
```

Comment:
disabling the picker on alerts page, right now it is not possible to change the data view anyway since the pattern is not passed to the server side